distbench_utils.cc

```diff
@@ -743,10 +743,32 @@ GetCanonicalDistributedSystemDescription(
   return ret;
 }
 
+absl::Status ValidateTestsSetting(const TestsSetting& settings) {
+  if (!settings.has_keep_instance_log()) {
+    return absl::InvalidArgumentError(
+        "TestsSettings does not have an option for keep_instance_log");
```

Comment:
it's ok not to have one.....

Reply:
What should I validate from the test settings?